src/presenters/business/business-email-change-presenter.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Formats data ready for presenting in the `/business-details` page
+ * @module businessEmailChangePresenter
+ */
+
+const businessEmailChangePresenter = (data, yar) => {
+  return {
+    notification: yar ? yar.flash('notification')[0] : null,
+    pageTitle: 'View and update your business details',
+    metaDescription: 'View and change the details for your business.',
+    businessEmail: data.changeBusinessEmail
+  }
+}
```

Comment:
do you need Business name, SBI and Username for the header in here?

Reply:
Yes, I need them. Added and refactored business-email-change.njk